torchtext/csrc/vectors.cpp

```diff
@@ -43,6 +58,12 @@ struct Vectors : torch::CustomClassHolder {
     if (item != stovec_.end()) {
       return item->value();
     }
+    const auto &item_index = stoindex_.find(token);
```

Comment:
This change deserves verifying that there's no regression in performance.

Reply:
Sure, I will rerun our benchmark